python/pfs/drp/stella/reduceArcTask.py

```diff
@@ -76,6 +75,18 @@ def run(self, expRefList, butler, wLenFile=None, lineList=None, immediate=True):
         self.log.info('wLenFile = %s' % wLenFile)
         self.log.info('lineList = %s' % lineList)
 
+        """ read wavelength file """
+        wavelengthFile = readWavelengthFile(wLenFile)
+        traceIds = wavelengthFile['traceIds']
+        wavelengths = wavelengthFile['wavelengths']
+        xCenters = wavelengthFile['xCenters']
+        yCenters = wavelengthFile['yCenters']
+        traceIdsUnique = np.unique(traceIds)
```

Comment:
This unpacking is tiresome. I comment on this below, but you could write this much more concisely if `readWavelengthFile` returned a `NamedTuple`.